pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -550,7 +550,10 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 	podUID := string(pod.UID)
 	contName := container.Name
 	allocatedDevicesUpdated := false
-	for k, v := range container.Resources.Limits {
+	// NOTE: Skipping the Resources.Lists is safe here because:
+	// 1. If container Spec mentions Limits only, implicitly Requests, equal to Limits, get added to Spec.
+	// 2. If container Spec mentions Limits, which are greater than or less than Requests, will fail at validation.
```

Comment:
Sorry, I can't recall we have validation to verify 2) , `limits` can be greater than `requests` by design in scheduler ... or I misunderstood the comment here?